doc/source/data-ingest.rst

```diff
@@ -0,0 +1,140 @@
+.. _data-ingest:
+
+.. toctree::
+    :hidden:
+
+    conduce-entities
+
+====================
+Intro to Data Ingest
+====================
+
+Visualizing your data in Conduce begins with ingest.  Ingest is the process translating your data into a format that Conduce understands.  Conduce uses entities to define unique objects.  Entities are ingested into datasets.  Datasets hold collections of related entities and provide the interface a user needs to connect their data to a visualization renderer.
+
+Conduce needs a few key pieces of information in order to visualize your data in space and time, those are:
+
+**Identity**
+   Identities distinguish one entity from another.  An entity exists in Conduce as one more more samples.  All samples that describe the same entity share an identity.
```

Comment:
extra more in `more more`

Reply:
and the first "more" should be an "or"

Reply:
The user doesn't know what a sample is at this point

Reply:
RE sample definition, I'll think about that.